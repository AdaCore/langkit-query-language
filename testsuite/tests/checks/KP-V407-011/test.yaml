driver: 'checker'
rule_name: KP_V407_011
project: 'prj.gpr'
