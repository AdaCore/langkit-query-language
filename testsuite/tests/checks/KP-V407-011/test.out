main_pkg.adb:17:17: rule violation: possible occurrence of KP V407-011
17 |         (A   => This.M_Impl_A'Access,   --  FLAG
   |                 ^^^^^^^^^^^^^^^^^^^^

main_pkg.adb:18:17: rule violation: possible occurrence of KP V407-011
18 |          B   => This.M_Impl_A'Access);  --  FLAG
   |                 ^^^^^^^^^^^^^^^^^^^^

